cypress/e2e/login.cy.js

```diff
@@ -1,11 +1,54 @@
-/// <reference types="cypress" />
+
 
 describe('Login page', () => {
   beforeEach(() => {
-    
+    cy.visit('login');
+  });
+
+  it('Login with valid creds (tomsmith/SuperSecretPassword!)', () => {
```

Comment:
the name of each of your `it` should continue the describe name

```suggestion
  it('should allow to log in with valid credentials', () => {
```